SQL/New_patches/2019-12-05-Remove_lorisMenuPermissions_for_data_release.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM LorisMenuPermissions WHERE PermID=(SELECT PermID FROM permissions WHERE code='data_release_upload');
+DELETE FROM LorisMenuPermissions WHERE PermID=(SELECT PermID FROM permissions WHERE code='data_release_edit_file_access');
```

Comment:
This should go to the master branch, we don't have SQL patches with bugfix releases because it's far too complicated to keep track of what needs to be done for upgrades that way.

Reply:
... i dont like that we can t issue bugfixes with patches... bugs can be in the database

Reply:
@ridz1208 Is it possible to modify the code causing this bug on bugfix and issue the SQL changes as more of a "cleanup" on master?

Reply:
there is no code really... it's the patch and the schema changes. its just SQL.

i think we just need a documented procedure on how to handle this